leapp.spec

```diff
@@ -46,16 +46,20 @@ Requires:   libvirt-python
 Requires:   nmap
 Requires:   sshpass
 Requires:   python-enum34
+Requires:   docker 
+Requires:   rsync
```

Comment:
We did agree on that we do not depend on docker or rsync and don't want to install it.

We simply require target to have them - It's not a real dependency for the tool - Since it even supports doing things remotely